pkg/config/config.go

```diff
@@ -252,6 +329,56 @@ func newChannelGroup(channelConfig *Channel, mspConfig *mb.MSPConfig) (*cb.Confi
 	return channelGroup, nil
 }
 
+// newOrgConfigGroup returns an config group for a organization.
+// It defines the crypto material for the organization (its MSP).
+// It sets the mod_policy of all elements to "Admins".
+func newOrgConfigGroup(org *Organization, mspConfig *mb.MSPConfig) (*cb.ConfigGroup, error) {
+	orgGroup := newConfigGroup()
+	orgGroup.ModPolicy = AdminsPolicyKey
+
+	if org.SkipAsForeign {
+		return orgGroup, nil
+	}
```

Comment:
I still don't understand why this is part of this library. Why would we model this "feature" of the confgitxgen tool?